lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -325,6 +327,12 @@ module InternalConstants
                  :default => nil,
                  :secret => true
 
+    # The URL of Stackdriver Logging API. Right now this only works with the
+    # gRPC path (use_grpc = true).
+    # One common usage of this config is to provide a mocked / stubbed Logging
+    # API, e.g., http://localhost:52000.
```

Comment:
Probably worth mentioning that using `http://` will create an insecure channel.

Reply:
Added.